src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -0,0 +1,1876 @@
+//------------------------------------------------------------------------------
+/*
```

Comment:
I couldn't make heads or tails of the changes, so I split the commit. https://github.com/ximinez/rippled/commits/consensus

Now github can't make heads or tails of it, so I'll use `gitk`.